.github/workflows/ci-core-reusable.yml

```diff
@@ -349,23 +349,38 @@ jobs:
 
       - name: Run integration tests
         run: |
-          PASSED_ENV_VARS="RUN_CONTRACT_VERIFICATION_TEST" \
-          ci_run zk_supervisor test integration --no-deps --ignore-prerequisites --chain era &> ${{ env.INTEGRATION_TESTS_LOGS_DIR }}/rollup.log &
-          PID1=$!
 
-          ci_run zk_supervisor test integration --no-deps --ignore-prerequisites --chain validium &> ${{ env.INTEGRATION_TESTS_LOGS_DIR }}/validium.log &
-          PID2=$!
+          local chains=("era" "validium" "custom_token" "consensus")
+          local logs=(${{ env.INTEGRATION_TESTS_LOGS_DIR }} ${{ env.INTEGRATION_TESTS_LOGS_DIR }} ${{ env.INTEGRATION_TESTS_LOGS_DIR }} ${{ env.INTEGRATION_TESTS_LOGS_DIR }})
+          local pids=()
+          local statuses=()
```

Comment:
Now, I'm thinking what if we will have the command in `/bin` which is accepted 3 arguments: 
1. Command to run  `zk_supervisor test integration --no-deps --ignore-prerequisites`
2. Logs path
3. chains

In this way we would solve the problem with non local for shell pids and will remove tons of boilerplate

Reply:
Doing that right now!